tsconfig.json

```diff
@@ -34,13 +34,11 @@
     ]
   },
   "include": [
-    "custom-next-env.d.ts",
```

Comment:
no more need to go against best practice and override nextjs's types just to use svgs